src/data/reducers/assetsFilters.js

```diff
@@ -0,0 +1,20 @@
+import { FILTER_UPDATED } from '../actions/assets';
+
+const initialState = {
+  courseId: 'course-v1:edX+DemoX+Demo_Course',
+  page: 0,
+  images: false,
+  documents: false,
+  other: false,
+};
+
+const assetsFilters = (state = initialState, action) => {
```

Comment:
Could we give this guy a more generic name than `assetsFilters`? `images`, `documents`, and `other` are definitely filters, but `courseId` and `page` less so. Maybe something like `assetsParameters`?